articles/active-directory-b2c/custom-policy-developer-notes.md

```diff
@@ -37,11 +37,11 @@ Azure Active Directory B2C [user flows and custom policies](user-flow-overview.m
 | [Sign-up and sign-in](add-sign-up-and-sign-in-policy.md) with email and password. | GA | GA| |
 | [Sign-up and sign-in](add-sign-up-and-sign-in-policy.md) with username and password.| GA | GA | |
 | [Profile editing flow](add-profile-editing-policy.md) | GA | GA | |
-| [Self-Service password reset](add-password-reset-policy.md) | GA| GA| |
-| [Force password reset](force-password-reset.md) | GA | NA | |
+| [Self-Service password reset](add-password-reset-policy.md) | GA| GA| Available in China cloud, but only for custom policies.
+| [Force password reset](force-password-reset.md) | GA | NA | Available in China cloud, but only for custom policies. |
 | [Phone sign-up and sign-in](phone-authentication-user-flows.md) | GA | GA | |
-| [Conditional Access and Identity Protection](conditional-access-user-flow.md) | GA | GA | Not available for SAML applications |
-| [Smart lockout](threat-management.md) | GA | GA | |
+| [Conditional Access and Identity Protection](conditional-access-user-flow.md) | GA | GA | Not available for SAML applications. Limited CA features are available in Mooncake cloud. Identity Protection is not available in China cloud. |
```

Comment:
```suggestion
| [Conditional Access and Identity Protection](conditional-access-user-flow.md) | GA | GA | Not available for SAML applications. Limited CA features are available in China cloud. Identity Protection is not available in China cloud. |
```

Reply:
@bernawy 

I'm told that China cloud was in preview, going to GA soon.  If that's the case, then these changes need to be updated when all other updates are ready.

Reply:
Ok, how soon is it? What do you want me to do? Abandon this PR?